composer.json

```diff
@@ -7,7 +7,7 @@
         "symfony/process": "^2.6|^3.0",
         "react/event-loop": "^0.4",
         "react/stream": "^0.4",
-        "react/http": "dev-master#cd15204bd15d106d7832c680e4fb0ca0ce2f5e30",
+        "react/http": "^0.4",
```

Comment:
installing is fine, but the httpkernel-adapter is not compatible with that version, which broke basically php-pm.